tf2onnx/onnx_opset/generator.py

```diff
@@ -242,6 +242,16 @@ def version_1(cls, ctx, node, **kwargs):
                       name=node.name, outputs=node.output,
                       shapes=shapes, dtypes=dtypes)
 
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        dtypes = node.output_dtypes
+        ctx.remove_node(node.name)
+        shape = ctx.make_node("Shape", node.input).output[0]
+        zero_tensor = helper.make_tensor("value", dtypes[0], [1], vals=[0])
+        ctx.make_node("ConstantOfShape", inputs=[shape],
+                      attr={'value': zero_tensor},
+                      name=node.name, outputs=node.output,
+                      dtypes=dtypes)
```

Comment:
Please add one more blank line after this method.

Reply:
Sure, added.